RESULTS.md

```diff
@@ -199,3 +199,123 @@ The results were very promising, reaching pro strength after about a week.
 
 Our models and training data can be found in [the GCS bucket
 here](https://console.cloud.google.com/storage/browser/minigo-pub/v3-9x9).
+
+
+### V5, 19x19s, March-April
+
+
+With our newly improved cluster tools, better monitoring, etc, we were pretty
+optimistic about our next attempt at 19x19.  Our evaluation matches -- pitting
+different models against each other to get a good ratings curve -- were
+automated during the v5 run, and a lot of new data analysis was made available
+during the run on the 'unofficial data site' cloudygo.com
+
+Unfortunately, progress stalled shortly after cutting the learning rate, and
+seemed to never recover.  Our three most useful indicators were our value net's
+train error and validation error, our value net error on a set of professional
+games (aka "figure 3"), and our selfplay rating as measured by our evaluation
```

Comment:
consider screenshots or linkify

Reply:
where to host screenshots on the github readme?  don't want to put them in the repo.